Playmate.xcworkspace/xcuserdata/jocelyntseng.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -7,16 +7,16 @@
       <BreakpointProxy
          BreakpointExtensionID = "Xcode.Breakpoint.FileBreakpoint">
          <BreakpointContent
-            uuid = "FA558A54-9B2E-464D-ACC9-CFEBC14B7ABC"
+            uuid = "A4581BE0-F8CE-4568-B020-8C504512690D"
```

Comment:
I think we should try to add this file to .gitignore, so it is not in the repo anymore.